it-and-security/lib/linux/scripts/fedora-debian-enable-fleet-desktop.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Script assumes one user is using the desktop environment (no multi-session).
+# It was tested on Fedora 38, 39 and Debian 12.
+
+set -x
+
+run_uid=$(id -u)
+
+# Start detached script and exit as root (to send result back to Fleet).
+if [ $run_uid == 0 ] && [ $# -eq 0 ]; then
+	/bin/bash -c "/bin/bash $0 1 >/var/log/orbit/appindicator_script.log 2>/var/log/orbit/appindicator_script.log </dev/null &"
```

Comment:
The `1` here in `/bin/bash $0 1` is just to make `[ $# -eq 0]` fail when this runs as a background job?  It could be worth it to make that slightly more explicit just to spare our future brains.